demos/encrypted-vm-image/README.md

```diff
@@ -0,0 +1,120 @@
+# Trusted startup of Encrypted VM image
+
+This demo shows how one can start up a [confidential virtual machine (CVM)](https://cloud.google.com/compute/confidential-vm/docs/about-cvm) whose VM image is protected with JinDisk. It provides tools that (1) can convert a plain virtual machine (VM) image into a JinDisk-protected one and (2) can use a JinDisk-protected VM image to start up a confidential VM. During the VM startup, a customizable process of remote attestation is triggered to fetch the root key that can unlock the JinDisk-protected VM image.
+
+The demo presents how to build and deploy a JinDisk-encrypted image on both Intel TDX and AMD SEV, while users can experiment with the demo without any VM-based TEE support.
```

Comment:
This paragraph reads like it is another summary paragraph. I suggest rewriting it in the following way.

```
This demo is intended for TEE VMs, including Intel TDX and AMD SEV, although it is also runnable on non-TEE VMs (with weaker security guarantees, of course) .
```